e2e/hive/docker-compose-atlas.yml

```diff
@@ -0,0 +1,92 @@
+version: '3'
```

Comment:
Do we want to add this file to the repo?

Reply:
yes why not?